aiidalab_widgets_base/structures.py

```diff
@@ -27,13 +29,13 @@ class StructureManagerWidget(ipw.VBox):
         node_class(str): trait that contains structure_node type (as string).
     '''
 
-    structure = Union([Instance(Atoms), Instance(Data)], allow_none=True)
-    structure_node = Instance(Data, allow_none=True, read_only=True)
+    structure = Instance(Atoms, allow_none=True)
+    structure_node = Instance(Data, allow_none=True)
     node_class = Unicode()
 
-    DATA_FORMATS = {'CifData': 'cif', 'StructureData': 'structure'}
+    SUPPORTED_DATA_FORMATS = {'CifData': 'cif', 'StructureData': 'structure'}
 
-    def __init__(self, importers, storable=True, node_class=None, **kwargs):
+    def __init__(self, importers, storable=True, node_class=None, **kwargs):  # pylint: disable=too-many-branches
```

Comment:
Any way we could refactor the class structure to reduce the number of branches and thus complexity?

Reply:
This was only to test having modifiable both: `structure` and `structure_node` (02f5b206d2536b600da59736c1cf9329527489ae). As we agreed to revert that commit - things have become easier again.